tests/prague/eip7692_eof_v1/eip7620_eof_create/test_eofcreate.py

```diff
@@ -590,3 +592,82 @@ def test_eofcreate_invalid_index(
         ),
         expect_exception=EOFException.INVALID_CONTAINER_SECTION_INDEX,
     )
+
+
+@pytest.mark.parametrize(
+    ["destination_code", "expected_result"],
+    [
+        pytest.param(Op.ADDRESS, "destination"),
+        pytest.param(Op.CALLER, "caller"),
+        pytest.param(Op.CALLVALUE, "eofcreate_value"),
+        pytest.param(Op.ORIGIN, "sender"),
+    ],
+)
+def test_eofcreate_context(
```

Comment:
We could add a parameter to test using either a EOF factory (current implementation) or a contract-creating tx.

Reply:
yup, I thought about this too. I'll add this to my list, but that would belong in a different folder (7698) so maybe a separate PR

Reply:
@marioevz Meanwhile, I tacked on the 7698-context-vars test onto this branch, kicking off the 7698 tests. PTAL and let me know if this is ok to go (see also our other thread).